rpm/pext.spec

```diff
@@ -0,0 +1,44 @@
+#Disable debug packages
+%define debug_package %{nil}
+
+Name:	pext
+Version:	0.9
+Release:	1%{?dist}
+Summary:	Python-based extendable tool
+
+License:	GPLv3
+URL:		https://github.com/Pext/Pext
+Source0: 	https://github.com/Pext/Pext/archive/v%{version}.zip
+#This removes the dependency since dependency checks are done by dnf/yum/rpm.
+#Also in Fedora the module name is PyQt5
+Patch0:		remove-dependency.patch
+
+BuildRequires:	python3-devel
+Requires:	python3-qt5 python3-pyopengl
```

Comment:
I would think this would need python3-pygit2 too, because Pext does use that to manage module updates.

Also, what about libnotify and qt5-qtquickcontrols?